code/game/gamemodes/dynamic/dynamic_rulesets_midround.dm

```diff
@@ -570,17 +569,20 @@
 	cost = 10
 	requirements = list(101,101,101,80,60,50,30,20,10,10)
 	repeatable = TRUE
-	var/list/spawn_locs = list()
+	var/datum/team/abductor_team/new_team
 
 /datum/dynamic_ruleset/midround/from_ghosts/abductors/ready(forced = FALSE)
 	if (required_candidates > (dead_players.len + list_observers.len))
 		return FALSE
+	new_team = new
+	if(new_team.team_number > 4)
```

Comment:
Where does this number come from?

Reply:
Its the amount of abductor ufos on the z-level where they're stored.  If we ever exceed that, we ran out of places to put new abductors.

Reply:
Turn it into a define, then, or calculate it dynamically.